README.md

```
@@ -17,19 +17,18 @@ rand = "0.9"
 iroh = "*"
 iroh-gossip = "*"
 
-distributed-topic-tracker = "0.2"
+distributed-topic-tracker = "0.3"
 ```
 
 Basic iroh-gossip integration:
-
 ```rust,no_run
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)